.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Task
+about: When a user story has to be broken into smaller engineering chunks, use a task!
+title: 'Task: Part of #Issue'
+labels: ''
+assignees: Sgtpluck
+
+---
+
+This task is part of #71
+
+Full User Story:
+As a Program Specialist
+I want my new or urgent complaints to have a color coding
+So that I know what complaints have an urgent need to be addressed
+
+This task is scoped to:
+- [] 
```

Comment:
[suggestion] if you are adding a real user story as an example, you could consider adding an example task below, too, if you want to